app/styles/layout/_select-inline.scss

```diff
@@ -204,6 +204,7 @@ $list-height: 220px;
       margin: 5px 7px 0 0;
       position: absolute;
       width: $icon-width;
+      left: 10px;
```

Comment:
@filipecrosk we just do alphabetically for the sake of newcomers to the project. It's the only foolproof ordering method I've found here.